framework/requirements.txt

```
@@ -64,6 +68,8 @@ six==1.14.0
 SQLAlchemy==1.3.11
 swagger-ui-bundle==0.0.3
 tabulate==0.8.9
+typing-extensions==3.10.0.2
+typing_inspect==0.7.1
```

Should this be a hyphen instead of an underscore?
```suggestion
typing-inspect==0.7.1
```